packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.tsx

```diff
@@ -180,7 +184,7 @@ const CreateRancher: React.FC<CreateRancherProps> = ({
         className="my-6"
       >
         <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
-          {t('createRancherInfoMessage')} <br />
+          <Trans> {t('createRancherInfoMessage')}</Trans> <br />
```

Comment:
Trans component is deprecated https://react.i18next.com/legacy-v9/trans-component